webapps-backup/src/main/java/io/camunda/webapps/backup/repository/elasticsearch/ElasticsearchBackupRepository.java

```diff
@@ -269,7 +271,15 @@ public void executeSnapshotting(
             final var response = esClient.snapshot().create(request);
             listener.onResponse(response);
           } catch (final Exception e) {
-            listener.onFailure(e);
+            if (isErrorType(e, INDEX_MISSING_EXCEPTION_TYPE) && !onlyRequired) {
+              // Retry taking only the required indices
+              LOGGER.info(
```

Comment:
:wrench: Is this useful information for the user? It means I will be logging this _every time_ if I never deploy Optimize, right? I would downgrade as debug (and I guess this highlights the limitations of this approach).